etc/scripts/json2csv.py

```diff
@@ -114,9 +114,6 @@ def collect_keys(mapping, key_group):
             if not path.endswith('/'):
                 path = path + '/'
 
-        # alway create a root directory
-        path = '/code' + path
```

Comment:
Do you want me to put a display message such as "Do you want to prefix the path"?